src/main/java/org/broadinstitute/consent/http/db/DarCollectionSummaryDAO.java

```diff
@@ -140,7 +140,8 @@ List<DarCollectionSummary> getDarCollectionSummariesForSO(
       "ON dar.reference_id = dd.reference_id " +
     "WHERE c.create_user_id = :userId " +
     	"AND (e.latest = e.electionid OR e.electionid IS NULL) " +
-        "AND (LOWER(data->>'status') != 'archived' OR data->>'status' IS NULL ) "
+        "AND (LOWER(data->>'status') != 'archived' OR data->>'status' IS NULL ) " +
+        "AND ("+DarCollectionSummary.FILTER_DRAFT_DAR_COLLECTIONS+")"
```

Comment:
I'm kind of feeling like I would rather see the actual string here, just because the rest of the query is here too. Is the `FILTER_DRAFT_DAR_COLLECTION` variable being referenced anywhere else?

Reply:
I lean towards this feedback as well. We do have a pattern of extracting large query blocks reused across many queries, but this is only used once and it feels more readable as a string value here.